modules/ROOT/pages/reconnection-strategy-about.adoc

```diff
@@ -4,12 +4,16 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint, studio, reconnection strategies, reconnection strategy, retry policies, retry
 
-When a Mule app starts, a connectivity test runs on components, such as the HTTP or FTP connector, that need to connect to an external server. If the test fails, the default behavior is to log a warning message and continue with the deployment of the app, instead of blocking the deployment.
+When an operation in a Mule application fails to connect to an external server, the default is for the operation to fail immediately. Upon failing, the operation returns a connectivity error. For example, if an operation in Anypoint Connector for FTP (FTP Connector) fails to connect to an FTP server, the default is to throw an `FTP:CONNECTIVITY` error. The error name prepends a namespace that is specific to the connector, such as `FTP:CONNECTIVITY`, `HTTP:CONNECTIVITY`, or `DB:CONNECTIVITY` (for Anypoint Connector for Database (Database Connector).
```

Comment:
At the end of this paragraph is missing the closing parenthesis 
(for Anypoint Connector for Database (Database Connector) > (for Anypoint Connector for Database (Database Connector))
or you can remove the start parenthesis before `for Anypoint Connector for Database`